locust/web.py

```diff
@@ -40,7 +40,8 @@ def index():
         is_distributed=is_distributed,
         slave_count=slave_count,
         user_count=runners.locust_runner.user_count,
-        version=version
+        version=version,
+        host=runners.locust_runner.host or runners.locust_runner.locust_classes[0].host
```

Comment:
I like this PR, but this line needs to be fixed.  It currently causes a test failure.  See travis-ci logs for details